firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -195,8 +197,20 @@ class AppSettingsTableViewController: SettingsTableViewController,
             studiesSetting.updateSetting(for: value)
         }
 
-        // Only add this toggle to the Settings if Terms Of Service feature flag is enabled
+        // Only add this toggles to the Settings if Terms Of Service feature flag is enabled
         if isSentCrashReportsEnabled {
+            let sendTechnicalDataSettings = SendDataSetting(
```

Comment:
nit: I think you can just assign the variable directly to sendTechnicalDataSettings instead of having an intermediate let.

Reply:
We discussed in private. sendTechnicalDataSetting is a BoolSetting type and sendTechnicalDataSettings SendDataSetting